packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -582,12 +583,14 @@ class DatePicker extends React.Component {
     }
   }
 
-  setPreSelection = (date) => {
+  setPreSelection = (date,type,value) => {
     const isValidDateSelection = date ? isDayInRange(date, this.props.minDate, this.props.maxDate) : true
     if (isValidDateSelection) {
       this.setState({
         preSelection: date
       })
+      type === "month" ? this.updateAriaLiveStatus(getMonthFromDate(date, this.props)) :
+      type === "year" ? this.updateAriaLiveStatus(value, this.props) :
```

Comment:
```suggestion
      type === "year" ? this.updateAriaLiveStatus(value) :
```

Reply:
Yes ,sorry missed it https://github.com/cerner/terra-framework/pull/2036/commits/81d4628685d0d0aa75e88c775417002540d9538c